packages/components/src/internal/util/utils.ts

```diff
@@ -716,3 +716,70 @@ export const handleFileInputChange = (
         }
     };
 };
+
+
+export function parseCsvString(value: string, delimiter: string, removeQuotes?: boolean) : string[]
```

Comment:
nit: Since most of these usages seem to be passing a delimiter of `','` consider the following paradigm:

```ts
export function parseCsvString(value: string, removeQuotes?: boolean): string[] {
    return parseDelimitedString(value, ',', removeQuotes);
}

export function parseDelimitedString(value: string, delimiter: string, removeQuotes?: boolean): string[] {
    // ...
}
```

Reply:
I considered that (actually had it), but it didn't seem worth it really.